config/schema/og.schema.yml

```diff
@@ -1,3 +1,22 @@
+field.storage_settings.og_membership_reference:
+  type: mapping
+  label: 'Organic Groupds reference field storage settings'
+  mapping:
+    target_type:
+      type: string
+      label: 'Type of entity to reference'
+
+field.field_settings.og_membership_reference:
+  type: mapping
+  label: 'Organic Groupds reference field settings'
+  mapping:
+    handler:
+      type: string
+      label: 'Reference method'
+    handler_settings:
+      type: entity_reference_selection.[%parent.handler]
```

Comment:
I didn't find in ER as-well- Where is the `parent.handler` value set ?

Plus admittedly I don't really understand what this part is doing (but I see it in ER so, it seems right)